packages/desktop/src/renderer/store/create.ts

```diff
@@ -9,7 +10,17 @@ import { composeWithDevTools } from 'redux-devtools-extension'
 import reducers from './reducers'
 import { errorsMiddleware } from './middlewares'
 
-const sagaMiddleware = createSagaMiddleware()
+Sentry.init({
+  dsn: 'https://1ca88607c3d14e15b36cb2cfd5f16d68@o1060867.ingest.sentry.io/6050774',
+  environment: process.env.TEST_MODE ? 'test' : 'production'
```

Comment:
What it does exactly? Afaik we want to enable Sentry only for builds of type 'prepublish'

Reply:
I didn't know that, I've reused the old Sentry setup, @EmiM told me that this is just a tag to distinguish if someone run it locally

Reply:
We do use it on prepublish but it was in case someone was 'testing' sentry or TEST_MODE locally